src/stories/Project.stories.js

```diff
@@ -8,28 +8,36 @@ export default {
 }
 export const LongDescription = () => (
   <Project
-    tags={['Social Business', 'In-Person-Support', 'Non-Profit']}
-    title='Take a breather'
-    country='AUS'
-    description='Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis tempus erat at urna semper tempus. Cras euismod semper turpis, at luctus elit dignissim non. Curabitur id arcu in justo pellentesque.'
-    image='/images/wildfire.jpg'
+    data={{
+      tags: ['Social Business', 'In-Person-Support', 'Non-Profit'],
+      title: 'Take a breather',
+      country: 'AUS',
+      description:
+        'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Duis tempus erat at urna semper tempus. Cras euismod semper turpis, at luctus elit dignissim non. Curabitur id arcu in justo pellentesque.',
+      image: '/images/wildfire.jpg',
```

Comment:
Perhaps you would like to include an "alt" attribute to your images? :)
This link may be useful: https://stackoverflow.com/questions/15471688/adding-alt-attribute-to-image-in-javascript

Reply:
Currently, all images get an empty string as the alt-attribute as I have no way of knowing what they are about due to them being selected at random from unsplash. Potentially I will let the user add a alt when creating the project